test/library/packages/ReplicatedVar/rcRemote.future

```diff
@@ -0,0 +1,3 @@
+bug
+ReplicatedVar.rcRemote fails to compile for a basic program
```

Comment:
@jabraham17 please put "ReplicatedVar.rcRemote fails to compile for a basic program" on the first line after `bug: `. I believe this is our format for .future files.